### PR TITLE
10.10.10..10 server address

### PR DESCRIPTION
**Pull request by georgejhunt**
Only tested on rpi4, 64bit, Apr 4 2022 version

### CODE REVIEW

**Comment by holta** [comment]
+10 to the Metric System!

To celebrate 10 years!

Hopefully we can cut over (IIAB's default) from 172.18.96.1 to 10.10.10.10 in coming days, if no glitches arise.

- #3170

**Comment by jvonau** [comment]
#3264

File: roles/cups/tasks/install.yml [hunk 1]
```diff
@@ -105,7 +105,7 @@
 #     - "HostNameLookups On"    # More False Leads: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=530027
 #     - "ServerAlias *"
 #     - "#ServerName {{ iiab_hostname }}.{{ iiab_domain }}"    # box.lan
-#     - "#Listen {{ lan_ip }}:631"    # 172.18.96.1
+#     - "#Listen {{ lan_ip }}:631"    # {{ lan_ip }}
```
Comment (jvonau) [inline roles/cups/tasks/install.yml:108]: @holta likes to use the part to the right of the # as notes

File: roles/cups/templates/cups.conf.j2 [hunk 2]
```diff
@@ -21,7 +21,7 @@ location ~ ^/print(|/.*)$ {    # '~' -> '~*' for case-insensitive regex
         return 301 http://localhost:631;
     }
 
-    return 301 http://$host:631;   # For 192.168.0.x, 172.18.96.1, 10.8.0.y ETC
+    return 301 http://$host:631;   # For {{ lan_ip }}, 172.18.96.1, 10.8.0.y ETC
```
Comment (jvonau) [inline roles/cups/templates/cups.conf.j2:24]: @holta likes to use the part to the right of the # as notes, changed wrong ip address

File: roles/network/templates/dhcp/dhcpd-iiab.conf.j2 [hunk 3]
```diff
@@ -5,15 +5,22 @@ ddns-update-style interim;
 #ignore client-updates;
 
 option domain-name "{{ iiab_domain }}";
-option domain-name-servers      172.18.96.1;
-option ntp-servers		172.18.96.1;
+option domain-name-servers      {{ lan_ip }};
+option ntp-servers		{{ lan_ip }};
 
 subnet 172.18.96.0 netmask 255.255.224.0 {
```
Comment (jvonau) [inline roles/network/templates/dhcp/dhcpd-iiab.conf.j2:11]: The subnet here would be wrong

File: roles/network/templates/gateway/iiab-gen-iptables [hunk 4]
```diff
@@ -64,7 +64,7 @@ echo "iiab_gateway_enabled: $iiab_gateway_enabled"
 echo
 #network_mode=`grep iiab_network_mode_applied /etc/iiab/iiab.ini | gawk '{print $3}'`
 #echo -e "Network Mode: $network_mode\n"
-lan_ip=$(iiab_var_value lan_ip)    # 172.18.96.1
+lan_ip=$(iiab_var_value lan_ip)    # {{ lan_ip }}
```
Comment (jvonau) [inline roles/network/templates/gateway/iiab-gen-iptables:67]: @holta likes to use the part to the right of the # as notes

File: roles/nextcloud/README.md [hunk 5]
```diff
@@ -43,7 +43,7 @@ Useful PHP recommendations for these settings (while largely tailored to WordPre
 
 ## Using It
 
-Log in to Nextcloud at http://box/nextcloud, http://box.lan/nextcloud, http://172.18.96.1/nextcloud (or similar) using:
+Log in to Nextcloud at http://box/nextcloud, http://box.lan/nextcloud, http://{{ lan_ip }}/nextcloud (or similar) using:
```
Comment (jvonau) [inline roles/nextcloud/README.md:46]: This is not a template

File: roles/samba/templates/smb.conf.j2 [hunk 6]
```diff
@@ -92,7 +92,7 @@
 ;	netbios name = MYSERVER
 
 ;	interfaces = lo eth0 192.168.12.2/24 192.168.13.2/24
-	hosts allow = 127. 172.18.
+	hosts allow = 127. 172.18. 10.10.
```
Comment (jvonau) [inline roles/samba/templates/smb.conf.j2:95]: s/b 10.10.10 to match the subnet mask

**Comment by holta** [comment]
@georgejhunt can you add some-or-all of Jerry's above suggestions?

**Comment by holta** [comment]
Thanks @jvonau for the 6 excellent suggestions.

Hopefully @georgejhunt (and/or others) has time to help wrap up this PR so it can be merged shortly in the coming week.

IIAB needs to lose weight, removing deadweight from the OLPC era that was very well-meaning at the time, however some of which has long outlived its usefulness in this 3rd decade of our new millenium.  Time to move forward not backwards.

Excerpt from https://en.wikipedia.org/wiki/Private_network :

image

**Pull request merged**